docs/dependencies.md

```diff
@@ -0,0 +1,22 @@
+
+- sudo; passwordless config (needed even for root users)
+- iptables - k0sctl checks if iptables exists and installs it
+- curl - Same as above
+- uname - Present in all Unix and Linux distributions
+- command - 
+- chmod - expected to always be there
+- mktemp - expected to always be there
+- install - expected to always be there
+- grep - expected to always be there
+- mv - expected to always be there
+- rm - expected to always be there
+- cat - expected to always be there
+- hostname - expected to always be there
+- find - expected to always be there
+- du - expected to always be there
+- sed - expected to always be there
+- test
+- install
+- getenforce
+- mount - expected to always be there
+- slackpkg - Slackware provides OS support for Slackware Linux
```

Comment:
Description is a bit odd. Also missing rpm, zypper, dpkg, apk, apt, apt-get, ... for other distros.

Reply:
I don't see rpm, zypper, dpkg or apt in CTL codebase

Reply:
They're in rig's codebase.